lib/ruby/driver.rb

```diff
@@ -0,0 +1,111 @@
+require "ruby/driver/version"
+require 'active_support'
+require 'ripper'
+require 'yajl'
+
+module Ruby
+  module Driver
+      class MyException < StandardError
+      end
+
+      class RequestMessage
+          attr_reader :action
+          attr_reader :language
+          attr_reader :language_version
+          attr_reader :content
+          BAD_REQUEST = "Bad Request"
+          PARSE_AST = "ParseAST"
```

Comment:
language, language_version and action are no longer needed according to the newest driver spec.